packages/js-dash-sdk/src/SDK/Client/Platform/methods/contracts/history.ts

```diff
@@ -0,0 +1,61 @@
+// @ts-ignore
+import { DataContract, Identifier } from '@dashevo/wasm-dpp';
+import {
+  GetDataContractHistoryResponse,
+} from '@dashevo/dapi-client/lib/methods/platform/getDataContractHistory/GetDataContractHistoryResponse';
+import { Platform } from '../../Platform';
+
+const NotFoundError = require('@dashevo/dapi-client/lib/transport/GrpcTransport/errors/NotFoundError');
+
+declare type ContractIdentifier = string | Identifier;
+
+/**
+ * Get contracts from the platform
+ *
+ * @param {ContractIdentifier} identifier - identifier of the contract to fetch
+ * @param startAtMs
+ * @param limit
+ * @param offset
```

Comment:
Shall we define more high-level methods for SDK? Like get by block time and get by version and list all versions?

Reply:
We made a separate task